login.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+require_once 'connectToDB.php';
+require_once 'src/Models/UsersModel.php';
+require_once 'SessionHandler.php';
+
+session_start();
+
+function verifyLogin(): void
```

Comment:
why is this inside a function? your only using it once and calling it immediately